cfgov/v1/models/base.py

```diff
@@ -131,6 +131,14 @@ class CFGOVPage(Page):
         ObjectList(settings_panels, heading='Configuration'),
     ])
 
+    def get_authors(self):
+        """ Returns a sorted list of authors. Default is alphabetical """
+        return self.alphabetize_authors()
+
+    def alphabetize_authors(self):
+        """ Alphabetize authors of this page by last name """ 
+        return sorted(self.authors.names(), key=lambda x: x.split(" ")[-1])
```

Comment:
Thanks @vickumar1981 -  I'll update it to use `x.split()`. 

We can expect the names in a certain format, so I'm constraining the edge cases a little bit.  I'm not sure how important it is to order by first name when last names are identical, but that would be another specification; that's not an edge case. If the first names were also identical, then would we sort on middle name if there is one?  I guess I feel like we could overengineer these cases that aren't that common @schaferjh Want to chime in here?